cli/ceph/orch/orch.py

```diff
@@ -0,0 +1,82 @@
+from cli import Cli
+from cli.utilities.utils import build_cmd_from_args
+from utility.log import Log
+
+log = Log(__name__)
+
+
+class OrchApplyCommandError(Exception):
+    pass
+
+
+class Orch(Cli):
+    def __init__(self, nodes, base_cmd):
+        super(Orch, self).__init__(nodes)
+        self.base_cmd = f"{base_cmd} orch"
+
+    def ls(self, **kw):
+        """
+        Lists the status of the given services running in the Ceph cluster.
+
+        Args:
+            kw (dict): Key/value pairs that needs to be provided to the installer.
+
+            Example:
+                Supported keys:
+                    service_type (str): type of service (mon, osd, mgr, mds, rgw)(Optional).
```

Comment:
All parameters are optional here ?

Reply:
We pass optional parameters only as part of `kw` so no need to add keyord `(optional)`. Please remove.